src/dsp_tools/resources/schema/data.xsd

```diff
@@ -619,5 +656,23 @@
             <xs:selector xpath=".//*"/>
             <xs:field xpath="@ark"/>
         </xs:unique>
+        <!-- Define a key constraint for license IDs -->
```

Comment:
What are these key constraints used for? If I comment them out, I still get a warning if a wrong license/copyright ID is referenced.

Reply:
It is that it only accepts keys defined in those tags, if you don't define a key constraint any ID used in the file is acceptable.